### ISSUE TITLE
Add the actual url with an example to the docs?

### ISSUE THREAD

**Issue by cottrell** [jupyter/kernel_gateway#351]
I've used this before and can't remember what the actual url path is or where to find it. 

A very very specific CURL example should be at the top right after starting it. 

! curl something

from within the notebook.

**Comment by cottrell** [comment]
I think a commnet of look for the port and logging once you launch the process would help people save some time. 

```
[KernelGatewayApp] Registering resource: /hello/world, methods: (['GET'])
[KernelGatewayApp] Registering resource: /_api/spec/swagger.json, methods: (GET)
[KernelGatewayApp] The port 8888 is already in use, trying another port.
[KernelGatewayApp] Jupyter Kernel Gateway at http://127.0.0.1:8889
INFO:tornado.access:200 GET /_api/spec/swagger.json (127.0.0.1) 0.55ms
```

If somebody wants a doc contrib I can just do that.

**Comment by kevin-bates** [comment]
Hi @cottrell - this seems generally useful and a really good idea!  

I think adding a `curl` example from the shell rather than from a notebook would be more applicable since an appeal of the gateway projects is that they're not necessarily accessed by a Notebook/Lab front-end.  The command would be a quick sanity test that the webserver is responsive.

I would recommend using `GET` against the `api/kernelspecs` endpoint as that's non-intrusive and should return something.

Thank you for offering to do this.

**Comment by kevin-bates** [comment]
Fixed via #356
